vllm/transformers_utils/config.py

```diff
@@ -97,6 +99,14 @@ def get_config(
     return config
 
 
+def get_hf_image_processor_config(
+    model: Union[str, Path],
+    revision: Optional[str] = None,
+    **kwargs,
+) -> Dict:
```

Comment:
```suggestion
) -> Dict[str, Any]:
```

Need to also update the import

Reply:
Done.